integrationtest/hostmetrics_test.go

```diff
@@ -167,7 +167,8 @@ func loadObservedMetrics(t *testing.T, metricsJSONPath string) pmetric.Metrics {
 
 	t.Logf("Found %v bytes of data at %s, selecting %v bytes", len(data), metricsJSONPath, len(secondBatch))
 
-	metrics, err := pmetric.NewJSONUnmarshaler().UnmarshalMetrics(secondBatch)
+	unmashaller := &pmetric.JSONUnmarshaler{}
```

Comment:
nit: Looks to be a type, `unmarshaller` instead?

Reply:
Yup, typed to fast will fix.

Reply:
@braydonk Resolved.